### ISSUE TITLE
Can't make any browser work correctly with chat connect

### ISSUE THREAD

**Issue by brazenvoid** [QuickBlox/quickblox-javascript-sdk#375]
Am trying to implement the JS SDK with application session token from back-end. I have iOS and Android already integrated with little hassle. 

Our goal is to only make a very simple 1-1 video/audio chat option, no texts required. Once that is done, we can think of making it richer.

**Environment details**
*(Operating system, browser information, SDK version)*

- Windows 10
- Firefox 68
- Chrome 75
- Edge (latest)
- SDK v2.12.2

**Did this work before?**

Its my first time using QB with JS.

**Expected behavior**

It should connect or throw an error with some mitigation detailed somewhere.

**Actual behavior**

Endless XML parsing errors on Firefox and Edge. 

Chrome does not throw any error but also does not enter the connect callback and calling resolves to the opponent not getting any notification for it.

There are significant differences in headers when request is sent from Chrome and the rest like the following only get sent from Chrome;
```
Sec-WebSocket-Extensions: permessage-deflate; client_max_window_bits
Accept-Encoding: gzip, deflate, br
Accept-Language: en-US,en;q=0.9
Pragma: no-cache
```

**Logs**

No meaningful logs. Firefox and edge don't even progress to show any, while all is green in Chrome. Had to add my own to gain some clarity.

**Code**

A stripped down code snippet is attached:
[Sample.txt](https://github.com/QuickBlox/quickblox-javascript-sdk/files/3345767/Sample.txt)

**Comment by ghost** [comment]
Hello QuickBlox customer,

This is Nikolay from QuickBlox support.

The issue was closed as it was outdated.

Please check the relevant section of our documentation here:
[https://docs.quickblox.com/docs/js-video-calling](https://docs.quickblox.com/docs/js-video-calling)

Also, please update the SDK to the latest version:
[https://github.com/QuickBlox/quickblox-javascript-sdk/releases/tag/2.13.10](https://github.com/QuickBlox/quickblox-javascript-sdk/releases/tag/2.13.10)

Additionally, please check our new sample:
[https://github.com/QuickBlox/quickblox-javascript-sdk/tree/gh-pages/samples/webrtc](https://github.com/QuickBlox/quickblox-javascript-sdk/tree/gh-pages/samples/webrtc)

If it is still relevant after reviewing the updated information, feel free to open a new issue.

Have a nice day.
